ASPnetCoreMVC/Views/Cart/Cart.cshtml

```diff
@@ -65,6 +64,33 @@
                     }
                 });
             });
+            $(document).on('click', '.remove-cart', function (event) {
+                debugger;
+                event.preventDefault();
+                var productid = $(this).data('id');
```

Comment:
$(this). lucs nay chính là nút được chọn á, vì mình bind action cho 1 class thì nó có nhiều item được chọn.